research/rainbow_options_bruteforce_method/rainbow_options_bruteforce_method.ipynb

```diff
@@ -0,0 +1,671 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    #def round_factor(a):
</pre><p>is this one on purpose?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
It was definitely something that was left and had to be removed.
I'll push a commit soon with the fix.